lib/src/VideoFrameWrapper.cpp

```diff
@@ -15,7 +15,11 @@ uint64_t VideoFrameWrapper::GetBufferSize() const
         case PixelFormat::Interleaved8BitRGBA:
         case PixelFormat::Interleaved8BitBGRA:
         case PixelFormat::Interleaved8BitARGB: {
-            return static_cast<uint64_t>(width) * height * 4;
+            if (stride > 0) {
```

Comment:
Since every other case here assumes a non-zero stride, let's just do the same here.